filebeat/module/haproxy/_meta/fields.yml

```diff
@@ -7,121 +7,137 @@
+        - name: total_waiting_time_ms
+          description: Total time in milliseconds spent waiting in the various queues
           type: long
 
-        - name: time_queue
-          description: time_queue is the total time in milliseconds spent waiting in the various queues.
+        - name: connection_wait_time_ms
+          description: Total time in milliseconds spent waiting for the connection to establish to the final server
           type: long
 
-        - name: time_backend_connect
-          description: time_backend_connect is the total time in milliseconds spent waiting for the connection to establish to the final server, including retries.
+        - name: bytes_read
+          description: Total number of bytes transmitted to the client when the log is emitted.
           type: long
 
-        - name: time_server_response
-          description: time_server_response is the total time in milliseconds spent waiting for the server to send a full HTTP response, not counting data.
+        - name: time_queue
+          description: Total time in milliseconds spent waiting in the various queues.
           type: long
 
-        - name: time_duration
-          description: time_duration is the time the request remained active in haproxy, which is the total time in milliseconds elapsed between the first byte of the request was received and the last byte of response was sent.
+        - name: time_backend_connect
+          description: Total time in milliseconds spent waiting for the connection to establish to the final server, including retries.
           type: long
 
         - name: server_queue
-          description: server_queue is the total number of requests which were processed before this one in the server queue.
+          description: Total number of requests which were processed before this one in the server queue.
           type: long
 
         - name: backend_queue
-          description: backend_queue is the total number of requests which were processed before this one in the backend's global queue.
+          description: Total number of requests which were processed before this one in the backend's global queue.
           type: long
 
         - name: bind_name
-          description: bind_name is the name of the listening address which received the connection.
+          description: Name of the listening address which received the connection.
 
         - name: error_message
-          description: error_message is the error message logged by HAProxy in case of error.
+          description: Error message logged by HAProxy in case of error.
+          type: text
+          
+        - name: source
           type: text
+          description: The HAProxy source of the log
         
         - name: geoip
           type: group
           description: >
-            Contains GeoIP information gathered based on the client_ip field.
+            geoip contains information gathered based on the client.ip field.
```

Comment:
Oh, I didn't see this in the original PR :slightly_frowning_face:  This shouldn't have been changed, but let it for future PRs.

Reply:
Can't I change it now?

Reply:
Is it ok to leave it like this? `Contains GeoIP information gathered based on the client.ip field.` (with `client.ip` instead of `client_ip`)

Reply:
Oh, yes, leave it like it was before, but with `client.ip`.

Reply:
Sure, done here https://github.com/elastic/beats/pull/8691